src/esp/physics/bullet/BulletRigidObject.cpp

```diff
@@ -305,25 +305,27 @@ void BulletRigidObject::setCollisionFromBB() {
   }
 }  // setCollisionFromBB
 
-bool BulletRigidObject::setMotionType(MotionType mt) {
+void BulletRigidObject::setMotionType(MotionType mt) {
   if (mt == MotionType::UNDEFINED) {
-    return false;
+    LOG(WARNING) << "BulletRigidObject::setMotionType : Cannot set motion type "
```

Comment:
Shouldn't this be an ERROR level logging?

Reply:
I don't think so - this would only be accessed by a user directly attempting to set the motion type to an undefined value.  Originally, the process only returned "false" if attempting to set the motion type to UNDEFINED, and if the user was not explicitly checking for a return value, the issue was totally missed.